src/routes/lovecasts/index.json.ts

```diff
@@ -18,16 +20,23 @@ export const get: RequestHandler = async () => {
 				})
 		: fakeResponse;
 
-	const lovecastsData: ConvertedLovecast[] = lovecasts.data.map((lovecast) => {
-		const { id, name_of_the_set, design, soundcloud_link } = lovecast;
-		return {
-			id,
-			title: name_of_the_set,
-			soundcloud: soundcloud_link,
-			imageUrl: design ? `${import.meta.env.VITE_DIRECTUS_URL}assets/${design}` : null
-		};
-	});
-
+	const lovecastsData: ConvertedLovecast[] = lovecasts.data
+		.map((lovecast) => {
+			const { id, name_of_the_set, design, soundcloud_link, type } = lovecast;
+			const [_, regexCastType, episodeNumber] = name_of_the_set.match(episodeNumberRegex);
```

Comment:
This was the easiest, least effort way to get the episode number. We could also solve this on the database layer, but would be more initial effort.

Potentially, we could run a migration to populate existing lovecasts though. 🤔